### ISSUE TITLE
Blue skies mod incompatibility

### ISSUE THREAD

**Issue by NoEndMan** [Tape-Dispenser/Immersive-Music-Mod#33]
I am using a custom modpack for minecraft 1.20.1 forge 47.3.1 with Immersive Music 1.0.3 mod and Blue Skies mod 1.3.30. Upon entering one of Blue Skies mod dimensions, the game crushes with the error: "java.lang.NullPointerException: Cannot invoke "net.minecraft.client.resources.sounds.SoundInstance.m_8070_()" because "soundInstance" is null".

Steps i took in trying to fix this issue:
1. Changing Blue Skies mod version to anything available for 1.20.1 - any version didn't work
2. Changing Immersive Music mod from 1.0.3 to 1.0.4 - didn't work with same error
3. Removing Immersive Music mod - That error wasn't thrown and Blue Skies mod worked fine (solved)

Since step 3 has worked i am assuming the source is concerning this mod. Could you please check it out?

[latest.log](https://github.com/user-attachments/files/18263001/latest.log)

**Comment by Tape-Dispenser** [comment]
yeah there's some bad code in the song selection algorithm that should be fixed in the upcoming version of timm

for now can you send your biome_playlists.json please? it should be in .minecraft/config/TIMM/

**Comment by NoEndMan** [comment]
[biome_playlists.json](https://github.com/user-attachments/files/18909941/biome_playlists.json)

**Comment by NoEndMan** [comment]
btw this bug also repliacted with most custom dimensions mods like aether

**Comment by Tape-Dispenser** [comment]
interesting, it seemed like a commin issue we have but thats not the case. I'll have to do some digging. Thanks!
